test/suites/smoke-test/test-configuration-consistency.ts

```diff
@@ -45,5 +45,33 @@ describeSuite({
           }
         }
     });
+
+    it({
+      id: "C03",
+      title: "Config all registered paras should be filled if more than min collators in orchestrator",
+      test: async function () {
+        const config = await api.query.configuration.activeConfig();
+        // get current session
+        const sessionIndex = (await api.query.session.currentIndex()).toNumber();
+        // get current authorities
+        const authorities = (await api.query.authorityAssignment.collatorContainerChain(sessionIndex)).toJSON();
+        
+        // If we have container chain collators, is because we at least assigned min to orchestrator
+        if (Object.keys(authorities["orchestratorChain"]).length > config["minOrchestratorCollators"].toNumber()) {
+          let liveContainers = await api.query.registrar.registeredParaIds();
+
+          const pendingVerification = await api.query.registrar.pendingVerification();
+          liveContainers = liveContainers.filter( function( el ) {
+            return pendingVerification.indexOf( el ) < 0;
+          } );
```

Comment:
This filter is not needed, calling markValidForCollating moves the para id from PendingVerification to RegisteredParaIds, so it is not possible to have the same para id in PendingVerification and in RegisteredParaIds.